gdb-server/src/worker.rs

```diff
@@ -88,6 +88,12 @@ pub async fn handler(
                 Query(QueryPacket::HostInfo) => handlers::host_info(),
                 ReadGeneralRegister => handlers::read_general_registers(session.core(0)?),
                 ReadRegisterHex(register) => handlers::read_register(register, session.core(0)?),
+                WriteGeneralRegister { reg_values } => {
+                    handlers::write_general_registers(&reg_values, session.core(0)?)
+                }
+                crate::parser::Packet::WriteRegisterHex { address, value } => {
```

Comment:
Please use a `use` for this as well, to make it consistent with the other match arms.